src/language-html/utils.js

```diff
@@ -530,7 +530,15 @@ function getNodeCssStyleDisplay(node, options) {
       return "inline";
     case "ignore":
       return "block";
-    default:
+    default: {
+      // See https://github.com/prettier/prettier/issues/8151
+      if (
+        options.parser === "vue" &&
+        node.name === "template" &&
+        node.parent.type === "root"
+      ) {
```

Comment:
Do we need check `lang`?

Reply:
I think we don't need check `lang`. I'm not sure why it is needed.

Reply:
Ya, seems we don't need that.